src/templates/language_selector.html

```diff
@@ -1,4 +1,4 @@
-<ul>
+<div id="language-selector">
```

Comment:
Ah, I get it now (hadn't read it closely before, just noticed the tag change). That markup - mixing divs and lis as children of a ul - is rather sketchy. Two alternate thoughts:
- Only append the language selector if it isn't already there (add a flag id/class to the divider or the nav-header)
- Give each of these LIs a class and remove everything with that class in _makeLanguageSelectorDropdown